ballerina/tests/from_json_test.bal

```diff
@@ -1752,3 +1752,29 @@ function testReadonlyFieldsWithNameAnnotation() returns error? {
     ReadonlyFieldsRec27 r7 = check parseAsType(user);
     test:assertEquals(r7, {testId: 4012, testTaxNo: "N/A", testName: "John Doe"});
 }
+
+@test:Config
+function testMapAsExpectedTypeWithJsonSource() returns error? {
+    json jsonValue = {id: "chamil", values: {a: 2, b: 45, c: {x: "mnb", y: "uio"}}};
+
+    map<json> mapValue = check parseAsType(jsonValue, options = {allowDataProjection: false});
+    test:assertEquals(mapValue, {"id":"chamil","values":{"a":2,"b":45,"c":{"x":"mnb","y":"uio"}}});
```

Comment:
```suggestion
    test:assertEquals(mapValue, {"id": "chamil", "values": {"a": 2, "b": 45, "c": {"x": "mnb", "y": "uio"}}});
```

Reply:
Update in 907da11a2710ca5da519003e7920cad082df7733